.baseline/checkstyle/checkstyle-suppressions.xml

```diff
@@ -21,4 +21,7 @@
     <!-- Generated code should not be subjected to checkstyle. -->
     <suppress files="[/\\].*[/\\]generated_src[/\\]" checks="." />
     <suppress files="[/\\].*[/\\]generated_testSrc[/\\]" checks="." />
+
+    <!-- Exception for SunPkcs1Reader -->
+    <suppress files="SunPkcs1Reader.java" checks="IllegalImport" />
```

Comment:
Yeah I can start a discussion on the Baseline side about the broader issue. It would definitely be nicer to have the suppression noted in code instead of externally (both for legibility and so that the next Baseline upgrade doesn't clobber the modifications), and I think there is something to having a more general solution like adding a `SuppressWithNearbyCommentFilter` that emulates the `@SuppressWarnings` behavior to deal with situations like these -- we could literally construct the regex such that the format is the same, but is just commented instead of being an actual annotation.

Will open a proposal/PR for that on Baseline (and possibly apply that to the Baseline files here as well in the next PR for this)